src/steps/os/unix.rs

```diff
@@ -282,6 +301,25 @@ pub fn run_brew_formula(ctx: &ExecutionContext, variant: BrewVariant) -> Result<
         }
     }
 
+    #[cfg(target_os = "linux")]
+    {
+        let sudo_uid = brew_linux_sudo_uid();
+        // if brew is owned by another user, execute "sudo -Hu <uid> brew update"
+        if let Some(uid) = sudo_uid {
+            let sudo_uid_str = sudo_uid.map_or_else(|| "".to_string(), |uid| uid.to_string());
+            print_separator(format!("{} (sudo as {})", variant.step_title(), sudo_uid_str));
+            Command::new("sudo")
```

Comment:
We can use the `sudo` from `ctx`:

```suggestion
             let sudo = crate::util::require_option(ctx.sudo().as_ref(), "sudo is needed to run the update".into())?;
            Command::new(sudo)
```

Reply:
BTW, can we use the `ctx.run_type().execute(sudo)` instead of `Command::new()` here?